Apps/WebClient/src/ClientApp/src/models/encounter.ts

```diff
@@ -0,0 +1,18 @@
+import Clinic from "@/models/clinic";
+
+export default interface Encounter {
+    // Gets or sets the id.
+    id: string;
+
+    // Gets or sets the practitioner name.
+    practitionerName: string;
+
+    // Gets or sets the specialty description.
+    specialtyDescription: string;
+
+    // Gets or sets the encounter timeline datetime.
+    serviceDateTime: Date;
+
+    // Gets or sets the clinic.
+    clinic: Clinic;
```

Comment:
Do you know if this one could possibly be undefined? if we dont know it will always be populated. If we know is going to be always populated we should leave it as is

Reply:
I don't actually know that yet but I expect to always have a clinic/location